typings/index.d.ts

```diff
@@ -2784,19 +2784,14 @@ declare module 'discord.js' {
     [K in keyof Omit<
       T,
       'client' | 'createdAt' | 'createdTimestamp' | 'id' | 'partial' | 'fetch'
-    >]: // tslint:disable-next-line:ban-types
-    T[K] extends Function ? T[K] : T[K] | null;
+    >]: T[K] extends Function ? T[K] : T[K] | null; // tslint:disable-next-line:ban-types
```

Comment:
```suggestion
    >]: T[K] extends Function ? T[K] : T[K] | null; // tslint:disable-line:ban-types
```

Reply:
I'm pretty sure the linting in this was done during the commit, not something I applied myself, so I have no idea why it does meet it's own requirements.